app/assets/javascripts/libs/input.js

```diff
@@ -120,9 +127,16 @@ export class InputKeyboard {
   bindings: Array<KeyboardBindingDownUp> = [];
   isStarted: boolean = true;
   delay: number = 0;
+  supportInputElements: boolean;
```

Comment:
This is missing the default value of false I think.

Reply:
Nice catch! Thanks!